reference/mongodb/mongodb/driver/monitoring/serverheartbeatfailedevent/geterror.xml

```diff
@@ -10,7 +10,7 @@
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
-   <modifier>final</modifier> <modifier>public</modifier> <type>Exception</type><methodname>MongoDB\Driver\Monitoring\ServerHeartbeatFailedEvent::getError</methodname>
+   <modifier>final</modifier> <modifier>public</modifier> <type>Throwable</type><methodname>MongoDB\Driver\Monitoring\ServerHeartbeatFailedEvent::getError</methodname>
```

Comment:
Note: the protos for `ServerHeartbeatFailedEvent::getError` and `CommandFailedEvent::getError` still use `Exception`. I realize you used `Throwable` in the stubs, but maybe you'd like to go back and update the protos to be consistent.

No objection to using Throwable, but I think Exception was also technically correct. I can't imagine a PHP error ever causing a command or heartbeat failure. These are all going to be actual exceptions derived from libmongoc errors.

Reply:
Good point. I've changed this back here and in the stubs, as we won't return any throwable but rather an exception.